sn_cli/src/subcommands/files.rs

```diff
@@ -272,6 +266,18 @@ pub(crate) async fn files_cmds(
     Ok(())
 }
 
+fn file_count(file_path: &PathBuf) -> u32 {
```

Comment:
can we add some docs here please.

Ie, does this recursively count files in a dir and child folders?